path-manager/app/AppLoader.scala

```diff
@@ -0,0 +1,29 @@
+import controllers._
+import play.api._
+import play.api.ApplicationLoader.Context
+import play.api.routing.Router
+import play.filters.HttpFiltersComponents
+import router.Routes
+import services.{KinesisLogging, Metrics}
+
+class AppLoader extends ApplicationLoader {
+  def load(context: Context) = {
+
```

Comment:
Pretty sure you need to initialise logging around here somewhere...
https://www.playframework.com/documentation/2.8.x/ScalaCompileTimeDependencyInjection#Configuring-Logging

Reply:
yeah its on the line below this one already ⬇️ 
https://github.com/guardian/path-manager/pull/32/files/fece6cb2bb0c2a9e2e44420d7977d79fcc441db2#diff-772c3661b87aa536d99a88385fe8ddc9R12

Reply:
and it works in CODE